liquidprompt

```diff
@@ -1183,6 +1185,31 @@ _lp_ruby_env_color() {
     lp_ruby_env_color="[${LP_COLOR_VIRTUALENV}${lp_ruby_env}${NO_COL}]"
 }
 
+_lp_terraform_env() {
+    (( LP_ENABLE_TERRAFORM )) || return 2
```

Comment:
This needs a check to disable this feature if `terraform` is not available. After line 493 in `lp_activate()`, add
```bash
    _lp_require_tool TERRAFORM terraform
```